devtools/common/src/test/java/io/quarkus/cli/commands/CreateProjectTest.java

```diff
@@ -27,6 +24,7 @@
 import java.util.zip.ZipOutputStream;
 
 import org.apache.maven.model.Model;
+import org.assertj.core.util.Sets;
```

Comment:
Is it really the class you want to use? I think it should just be a `new HashSet<>()`.

Reply:
yes, I didn't really want that class